server/schedulers/hot_test.go

```diff
@@ -412,7 +413,7 @@ func (s *testHotReadRegionSchedulerSuite) TestWithPendingInfluence(c *C) {
 	//|    1     |       7.1MB      |
 	//|    2     |       6.1MB      |
 	//|    3     |        6MB       |
-	//|    4     |        5MB       |
+	//|    4     |        4MB       |
```

Comment:
Why change this? The data below is not changed.

Reply:
In order to pass the variance test, the following data has been modified. I forgot to recover the comments above, thank you.